buildenv/jenkins/variables/defaults.yml

```diff
@@ -347,9 +347,9 @@ win_x86-64_cmprssptrs:
       next: 'hw.arch.x86 && sw.os.windows'
   build_env:
     vars:
-      8: 'PATH+TOOLS=/cygdrive/c/openjdk/nasm-2.13.03/'
-      11: 'PATH+TOOLS=/cygdrive/c/openjdk/nasm-2.13.03/:/cygdrive/c/mingw-w64/x86_64-8.1.0-win32-seh-rt_v6-rev0/mingw64/bin/'
-      next: 'PATH+TOOLS=/cygdrive/c/openjdk/nasm-2.13.03/:/cygdrive/c/mingw-w64/x86_64-8.1.0-win32-seh-rt_v6-rev0/mingw64/bin/'
+      8: 'PATH+TOOLS=/cygdrive/c/openjdk/LLVM64/bin/:/cygdrive/c/openjdk/nasm-2.13.03/'
```

Comment:
I think it is unwise to include the trailing `/` in elements of the path.
They should also be removed from the build instructions.

Reply:
Already removed.